.circleci/config.yml

```diff
@@ -304,7 +304,7 @@ workflows:
           requires: [ "Build Tests" ]
       # Always run UI tests on develop and release branches
       - UI Tests:
-          name: UI Tests (iPhone 11)
+          name: UI Tests (iPhone)
```

Comment:
Removed the device model information, so we don't have to keep updating it. 

I think there's value in knowing the tests are on iPhone or iPad, but given we don't run on multiple iPhone (or iPad) Simulators, knowing the exact device is not necessary.

This is also the pattern already adopted in the Buildkite pipeline: https://github.com/wordpress-mobile/WordPress-iOS/blob/ab81d4d401199b43da1af966596c734434ca4392/.buildkite/pipeline.yml#L47

Reply:
I wonder if there is a risk for some UI tests to pass on a given iPhone model but fail on another 🤔 I'm especially thinking about UI tests written such that they scroll the screen a certain amount hoping to put an element into view before tapping it, but with different screen sizes the amount would be different etc.

I'm sure for those kind of cases written that way we could rewrite the test to be more generic (eg "scroll in a while loop until the element is visible") so that they are more flexible. But just thought I'd ponder about this out loud anyway in case you had thoughts about it.

Reply:
Yes. I think there's a non trivial chance of this happening. At this point in time, though, we are basically running tests always on the same limited set of devices (e.g. @pachlava and I have been using the iPhone 12 Simulator for Simplenote because it's proven to be the most stable when he started writing tests).

I'd love to get to the point when we have a code coverage that's robust enough on one device that we can start thinking about testing on different ones. I'd be in particularly keen to run the tests on the smallest screen, biggest screen, and most common device on our user base.

Also, yes we ought to write tests in a "generic" way. And hopefully XCUITestHelpers will help us with that and with keeping in check issues across different devices.

_One day..._ 😄 

(I've been using devices as a synonym for Simulators)